modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/frontend/taglib/servlet/taglib/MFASetupUserAccountScreenNavigationEntry.java

```diff
@@ -43,6 +41,7 @@
  * @author Marta Medio
  */
 public class MFASetupUserAccountScreenNavigationEntry
+	extends MFASetupUserAccountScreenNavigationCategory
```

Comment:
@RenanUmV This is not a component, no need to extend the `Category`. In this case, the protected `Language` field is never initialized in the `Entry` instance, and it's actually not used either.